### PR TITLE
design order page

### PR DESCRIPTION
**Pull request by vandanghuy120385**
## Related Tickets:

Task #68145: tạo giao diện order  
Task #68147: Kết nối giao diện order vào controller
## WHAT this PR does?
Tao giao dien order va connect to Controller

## Notes Impacted Areas:
- app/http/controller
- resources/views

## Attachments:
- N/A.

### CODE REVIEW

**Comment by hoangnbvn** [comment]
ok

**Comment by BuiMinhDung711** [comment]
ok

**Comment by vandanghuy120385** [comment]
ready

File: app/Http/Controllers/OrderController.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                'shippingFee',
+                'paymentMethods',
+                'totalPayment',
+            ),
+        );
+    }
+
+    public function calculateShippingFee($billTotal, $address)
+    {
+        return $billTotal * 0.5; // shipping fee = 50% bill
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]: put 50% into config

File: app/Http/Requests/PlaceOrderRequest.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        $user = auth()->user();
+        if ($user) {
+            return true;
+        }
+        return false;
```
Comment (ghost) [inline app/Http/Requests/PlaceOrderRequest.php]:
```suggestion
        $user = auth()->user();
        if ($user) {
            return true;
        }

        return false;
```

File: resources/views/cart/index.blade.php [hunk 3]
```diff
@@ -18,24 +18,34 @@
...
-                                        name="{{ $cartItem->name }}" 
-                                        price="{{ $cartItem->price }}" 
-                                        quantity="{{ $cartItem->quantity }}" 
-                                        media-link="{{ $cartItem->media_link }}" />
-                                @endforeach
-                                <div class="flex justify-between items-center mt-6 pt-6 border-t">
-                                    <div class="flex justify-center items-end">
-                                        <span class="text-sm font-medium text-gray-400 mr-1">{{ _('Subtotal') }}: $</span>
-                                        <span class="text-lg font-bold text-gray-800 " id="sub-total"> {{$subTotal}}</span>
+                                @if( $cartItems->count() > 0)
```
Comment (ghost) [inline resources/views/cart/index.blade.php]:
```suggestion
                                @if ($cartItems->count() > 0)
```

File: resources/views/order/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                                    <h2 class="text-xl font-medium pr-8">
+                                        {{ __('Payment method')}}
+                                    </h2>
+                                </div>
+                                <div class="col-span-2"></div>
+                                <div>
+                                    <select name="payment_method" id="payment_method">
+                                        @foreach($paymentMethods as $payment => $method)
+                                        <option value="{{ $payment }}">{{ __($method) }}</option>
+                                        @endforeach
```
Comment (ghost) [inline resources/views/order/index.blade.php]:
```suggestion
                                        @foreach ($paymentMethods as $payment => $method)
                                            <option value="{{ $payment }}">{{ __($method) }}</option>
                                        @endforeach
```

File: routes/web.php [hunk 5]
```diff
@@ -58,6 +67,6 @@
         'auth',
         'verified',
         'role:admin',
-]);
+    ]);
```
Comment (ghost) [inline routes/web.php]: why indent here?

**Comment by vandanghuy120385** [comment]
ready

**Pull request merged**